extension/ballastextension/memory_ballast.go

```diff
@@ -24,6 +24,8 @@ import (
 
 const megaBytes = 1024 * 1024
 
+var ballastSizeBytes uint64
```

Comment:
@mxiamxia probably in a separate PR, please make this a property on the extension not a global variable. The MemoryLimiter has access to all extensions via the Host and should read from there.